### PR TITLE
feat(blog): a32nx flight model

### PR DESCRIPTION
**Pull request by Valastiri**
## Summary of changes
Supports marketing for A32NX Flight Model changes / accuracy / improvements.

[Courtesy Link to Page](https://website-git-fork-valastiri-blog-flight-model-flybywire.vercel.app/notams/a32nx-flight-model/)

## TODO

- [x] Finalize Header
- [x] Add "pretty" tables
- [x] Proof
- [x] Review and Schedule

## Screenshots(if applicable)
< Provide screenshots of the affected areas if applicable to support your changes being made ensuring you show the full scope of the area changed and surrounding content, this allows for faster reviews on PRs. >

### CODE REVIEW

File: src/posts/a32nx-flight-model.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+authors:
+  - 'donbike'
+category: 'UPDATE'
+metaImage: '/img/notam-images/flypados3/flypad-hero.png'
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Overview
+
+We have gone to great lengths to ensure that the A32NX flight model is as realistic as it gets. Collaborating with our own A320 pilot staff at FlyByWire Simulations and testing each change religiously the flight model is based on matching the following:
```
[the remark this answers is not in the document]
Comment (Valastiri) [reply src/posts/a32nx-flight-model.md:13]: Excellent feedback. I'll work in the changes this weekend. 

As for the charts I'm still considering best way to style them but I agree with altering some from raw data to visual such as bar graph in the case of stopping distances.

**Comment by Valastiri** [comment]
Still working on the charts // will post an update within a couple days

File: src/posts/a32nx-flight-model.md [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+  - 'Valastiri'
+category: 'UPDATE'
+metaImage: '/img/flight-model-feature.png'
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Overview
+
+We have gone to great lengths to ensure that the A32NX flight model is as realistic as it gets. Collaborating with our own A320 pilot staff at FlyByWire Simulations and testing each change 
+meticulously, the flight model is based on matching the data and other information obtained from the A320NEO Airplane Flight Manual, Flight Crew Operating Manual, Flight Crew Techniques Manual,
```
Comment (kevinkiloechovictor) [inline src/posts/a32nx-flight-model.md:15]:
```suggestion
meticulously, we have adjusted the flight model to data and other information obtained from the A320NEO Airplane Flight Manual, Flight Crew Operating Manual, Flight Crew Techniques Manual, 
```

File: src/posts/a32nx-flight-model.md [hunk 3]
```diff
@@ -0,0 +1,178 @@
...
+metaAlt: 'FlyByWire Simulations'
+---
+
+## Overview
+
+We have gone to great lengths to ensure that the A32NX flight model is as realistic as it gets. Collaborating with our own A320 pilot staff at FlyByWire Simulations and testing each change 
+meticulously, the flight model is based on matching the data and other information obtained from the A320NEO Airplane Flight Manual, Flight Crew Operating Manual, Flight Crew Techniques Manual, 
+Quick Reference Handbook, FlySmart electronic flight bag, and our A320NEO piloting team.
+
+If you are a more data oriented reader, below you can find comparisons between the reference data and our tests in MSFS. We have included a plethora of tables and charts for you.
```
Comment (kevinkiloechovictor) [inline src/posts/a32nx-flight-model.md:18]:
```suggestion
If you are a more data-oriented reader, below you can find comparisons between the reference data and our tests in MSFS. We have included a plethora of tables and charts for you.
```

File: src/posts/a32nx-flight-model.md [hunk 4]
```diff
@@ -0,0 +1,178 @@
...
+
+Credits:
+
+- donbikes for the flight model work,
+- Taz for the engine modelling,
+- Davy for the simulation of the autobrake system.
+
+**Note:** Some of the smaller more data heavy images can be clicked on so you can view the data in full.
```
Comment (kevinkiloechovictor) [inline src/posts/a32nx-flight-model.md:26]:
```suggestion
**Note:** Some of the smaller, more data-heavy images can be clicked on so you can view the data in full.
```

File: src/posts/a32nx-flight-model.md [hunk 5]
```diff
@@ -0,0 +1,178 @@
...
+
+---
+
+## Work In Progress Tuning
+
+Constant improvements with realistic and often very recent data points from our pilot staff is where we believe the A32NX shines. Recent updates to our aircraft fixes issues with inadequate
```
Comment (kevinkiloechovictor) [inline src/posts/a32nx-flight-model.md:32]:
```suggestion
Constant improvement with realistic and often very recent data points from our pilot staff is where we believe the A32NX shines. Recent updates to our aircraft fix issues with inadequate 
```

**Review by kevinkiloechovictor — CHANGES_REQUESTED** [review]

File: src/posts/a32nx-flight-model.md [hunk 6]
```diff
@@ -0,0 +1,178 @@
...
+
+## Work In Progress Tuning
+
+Constant improvement with realistic and often very recent data points from our pilot staff is where we believe the A32NX shines. Recent updates to our aircraft fix issues with inadequate 
+takeoff climb performance after an engine failure at heavy gross weights. You can now practice those V1 engine cuts at 2nd segment climb-limited TOGW. Be sure when doing so to not delay gear 
+retraction, lower your pitch rate and pitch target, and apply appropriate rudder to minimize control and sideslip drag! (**Note:** This capability is currently only available in the Development 
+and Experimental versions of the A32NX).
+
+We are currently fine-tuning the landing gear drag to better match climb, cruise, and descent data for gear down operations. The results of our WIP tuning are included in the tables below. We 
+expect to add these changes to the A32NX soon - first in the Experimental version, followed by the Development version, and then our Stable releases.
```
Comment (frankkopp) [inline src/posts/a32nx-flight-model.md:38]: I would not phrase it like this as this gives the impression that Experimental is the first release where we publish new stuff. This is not the case!
We do testing of selected feature in Exp - which is basically Dev +selection of to-be-tested features

```suggestion
expect to add these changes to the A32NX soon - we will start testing these in the Experimental version, then once they are sufficiently tested and stable we will publish it to the Development version, and eventually to our our Stable releases.
```

**Review by frankkopp — CHANGES_REQUESTED** [review]
Just one comment otherwise LGTM

File: src/posts/a32nx-flight-model.md (hunk 6 above)
Comment (Valastiri) [reply src/posts/a32nx-flight-model.md:38]: great point -- slight issue with ur edit (duplicate "our") -- will commit manually in a bit. Thanks!

**Comment by Valastiri** [comment]
Discussed on Discord -- ignoring lint for now as technically all is fine as per Sabes and refactor v3 will address build issues.

**Review by donstim — COMMENTED** [review]
Not in favor of the change to say "we have adjusted..."  I think (hope) he is referring to our latest update of the flight model, but it can be interpreted that we "just" have a modified Asobo default A320neo flight model. I would rather be clear that it is "our" flight model. We've never done a flight model NOTAM before (to my recollection), so I think it appropriate to sort of summarize our flight model in general in this first paragraph, not just the latest set of updates.

**Pull request merged**